### PR TITLE
Condrashov Serghei third java task

### PR DESCRIPTION
**Pull request by SerjCo84**
Comleted please cheak

### CODE REVIEW

File: src/main/java/serghei_condrasov/BreakContinue.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+public class BreakContinue {
+
+    public static void main(String[] args) {
+
+        // 1. Create a program, which will ask user to enter a number, if number will be negative exit the loop.
+        Scanner scan = new Scanner(System.in);
+        while (true) {
+            System.out.println("Enter your number :");
+            int n = scan.nextInt();
+            if (!(n >= 0)) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/serghei_condrasov/BreakContinue.java:14]: make it easier if (n<0)

File: src/main/java/serghei_condrasov/BreakContinue.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+                break;
+            }
+        }
+
+        // 2. Create a program which will print all number from 1 to 10 except 3 and 7 using continue keyword.
+        for (int i = 1; i <= 10; i++) {
+            if (i == 3) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/serghei_condrasov/BreakContinue.java:21]: combine to one if with or operator

File: src/main/java/serghei_condrasov/For.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        // 3. Calculate the factorial of a given number using a for loop.
+        int n = 4;
+        int result = 1;
+        for (int c = 1; c <= n; c++) {
+            result = result * c;
+        }
+        System.out.print("\n" + result);
+
+        // 4. (Optional ) Print all numbers from 20 to 0 with step 2 ( 20, 18, 16, 14).
+        for (int f = 20; f > 0; f--) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/serghei_condrasov/For.java:29]: step should be 2, not one
in last section f-=2

File: src/main/java/serghei_condrasov/For.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            if (f % 2 == 0) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/serghei_condrasov/For.java:30]: remove this if

**Review by dgrimaliuc-crunchyroll — CHANGES_REQUESTED** [review]

File: src/main/java/serghei_condrasov/BreakContinue.java (hunk 1 above)
Comment (SerjCo84) [reply src/main/java/serghei_condrasov/BreakContinue.java]: corrected

File: src/main/java/serghei_condrasov/BreakContinue.java (hunk 2 above)
Comment (SerjCo84) [reply src/main/java/serghei_condrasov/BreakContinue.java]: corrected

File: src/main/java/serghei_condrasov/For.java (hunk 3 above)
Comment (SerjCo84) [reply src/main/java/serghei_condrasov/For.java]: corrected

File: src/main/java/serghei_condrasov/For.java (hunk 4 above)
Comment (SerjCo84) [reply src/main/java/serghei_condrasov/For.java]: corrected

**Review by dgrimaliuc-crunchyroll — APPROVED** [review]

**Pull request merged**